multi-node.sh

```diff
@@ -8,6 +8,7 @@ slave_value=${!slave_name}
 ami=($slave_value)
 NODES=2
 tmp_script=$(mktemp -p $WORKSPACE)
+output_dir=$(mktemp -d -p $WORKSPACE)
```

Comment:
This should be a change to the "Clean up xml parsing code" commit I think, same for single-node.sh

Reply:
I messed up the "Move tmp_script init to common.sh" patch, fixed.